customer-wallet/src/containers/Payment/index.js

```diff
@@ -0,0 +1,155 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+// components
+import { Card, CardHeader, Divider, CardContent } from '@material-ui/core'
+import { List, ListItem, ListItemIcon, ListItemText } from '@material-ui/core'
+import { Avatar, withStyles } from '@material-ui/core'
+import PaymentForm from './PaymentForm'
+// hoc
+import withFxRates from 'containers/Hoc/withFxRates'
+// redux
+import { connect } from 'react-redux'
+import { compose } from 'recompose'
+import { getCheckout } from 'store/modules/data/checkout'
+import { setRatingForCustomerWallet } from 'store/modules/components/CustomerWallet'
+import { submitPayment } from 'store/modules/data/payment'
+// icons
+import WalletIcon from '@material-ui/icons/AccountBalanceWallet'
+import VendorIcon from '@material-ui/icons/Shop'
+import PaymentDestinationIcon from '@material-ui/icons/ArrowForward'
+import MarketplaceIcon from '@material-ui/icons/Store'
+import LoadingIcon from '@material-ui/icons/Sync'
+import AmountIcon from '@material-ui/icons/Payment'
+import ReviewIcon from '@material-ui/icons/Star'
+import ErrorIcon from '@material-ui/icons/ErrorOutline'
+
+const style = {
+    card: {
+        margin: '30px'
+    }
+}
+
+const starCount = 5
+
+class Payment extends Component {
+
+    componentDidMount() {
+        this.props.getCheckout()
+    }
+
+    handleSubmit(data) {
+        //this.props.submitPayment()
+        const review = data.review || ''
+        const rating = this.props.rating
+        this.props.submitPayment({ review, rating })
+    }
+
+    render() {
+        const {
+            classes,
+            wallet,
+            checkout: {
+                data: popr,
+                loading: checkoutLoading,
+                error: checkoutError
+            },
+            rating,
+            setRating,
+            convertSatoshiToBits,
+            convertSatoshiToBTC,
+            convertFromBtcToUsd
+        } = this.props
+
+        const submitDisabled = !popr || checkoutLoading || checkoutError
+        const amountSatoshi = popr.amount
+        const amountBits = convertSatoshiToBits(amountSatoshi)
+        const amountBtc = convertSatoshiToBTC(amountSatoshi)
+        const amountUSD = convertFromBtcToUsd(amountBtc)
+        const amountText = `${amountUSD} USD | ${amountBtc} BTC | ${amountBits} bits | ${amountSatoshi} satoshi`
```

Comment:
Show only USD and BTC for now. We might need to introduce mBTC instead of bits in the future.